example/cumulus-tf/main.tf

```diff
@@ -122,7 +122,7 @@ module "cumulus" {
 
   cmr_search_client_config = var.cmr_search_client_config
 
-  cmr_oauth_provider = var.cmr_oauth_provider
+  cmr_oauth_provider = "launchpad"
```

Comment:
To be clear, we're saying that we aren't supporting deploying with EDL as a CMR authorization at all?

Reply:
I can put the cmr_oauth_provider back, but hopefully no one uses it.